login1/dbus_test.go

```diff
@@ -26,3 +26,16 @@ func TestNew(t *testing.T) {
 		t.Fatal(err)
 	}
 }
+
+// TestInhibit ensures that Inhibit works without errors.
+func TestInhibit(t *testing.T) {
+	conn, err := New()
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	_, err = conn.Inhibit("sleep", "Test program", "Testing inhibitor method", "delay")
```

Comment:
Er, sorry, can you revert this entirely? An unwanted side effect if people are testing locally!